Sources/Orbit/Components/Tag.swift

```diff
@@ -3,10 +3,11 @@ import SwiftUI
 /// Offers a label that can optionally be selected and unselected or removed.
 ///
 /// - Note: [Orbit definition](https://orbit.kiwi/components/tag/)
+/// - Important: Component can expand horizontally by using `idealSize` set to false.
```

Comment:
Maybe
> Component will expand horizontally if `idealSize` is set to `false`

?

Reply:
> Very nice!
> 
> Seeing stuff like <img alt="Screenshot 2022-12-31 at 22 09 25" width="263" src="https://user-images.githubusercontent.com/12349477/210155681-4485d350-4d4b-4b82-abb3-6a4d120ef33a.png"> ...shouldn't `.measured()` be limited to a single line?

Thx, updated with a single line limit and previews.